docs/tutorials/experimental-fits.md

```diff
@@ -0,0 +1,436 @@
+# Experimental data fits
+
+Dynamiqs enables rapid simulations and facilitates automatic gradient computation, making it highly suitable for data fitting tasks. This tutorial will guide you through the process of simulating and fitting real-world experimental data, step by step.
+
+## Context of the experiment
+
+!!! note Info
+    This part introduces the kind of data and process that we will fit in the notebook. If you are only interested in the code you can skip it.
+
+!!! example "Diving deep 🤿"
+    You can discover further details about this experiment in the original research paper from which this data was sourced [^1]
+
+The data we will work with are deflation curves. Our system is composed of a high lifetime cavity - the memory -  coupled to a lossy one - the buffer - in vacuum state. We denote $a$ (respectively $b$) the memory (respectively buffer) annihilation operator. The memory and the buffer are coupled through a two to one photon exchange mechanism $g_2 a^2 b^\dagger + h.c.$. The buffer is a highly-dissipative mode, that we model with the dissipator $D[\sqrt{\kappa_b} b]$. Both cavities have a finite temperature, expressed in thermal population.
+
+At the beginning of the experiment the memory is populated with a coherent state and the buffer is in thermal vacuum. We enable for a period of two photon exchange mechanism. During this period, pairs of photons leave the memory and are converted into a single buffer photon, that is then dissipated into the environment. The memory photon number $a^\dagger a$ is measured at different times and saved into the `data` variable. Our goal is to infer $g_2$ from the data.
+
+![Pulse sequence](/figs-docs/g2-pulse-sequence.png)
+_Pulse sequence of the experiment, taken from [^1]_
+
+[^1]:  [Quantum control of a cat-qubit with bit-flip times exceeding ten seconds, arxiv:2307.06617](https://arxiv.org/pdf/2307.06617.pdf)
+
+## Step by step solution
+
+We start by importing all necessary libraries:
+
+```python
+import dynamiqs as dq
+import jax
+import jax.numpy as jnp
+import matplotlib.pyplot as plt
+import numpy as np
+from tqdm import tqdm
+```
+
+Declare useful helpers to handle units:
+
+```python
+MHz = 2 * np.pi
+kHz = 2 * np.pi * 1e-3
+us = 1.0
+ns = 1.0e-3
+```
+
+Define physical values that describe our system (taken from [^1]):
+
+```python
+thermal_a, thermal_b = 0.1, 0.011
+kappa_a = 9.3e-3 * kHz
+kappa_b = 2.6 * MHz
+```
+
+Instanciate our bosonic annihilation operators:
+
+```python
```

Comment:
If you only have single-line comments about each of these things, I would suggest putting them all in a single code block, with python comments above. It's easier to read and to follow.